pkg/apis/flowcontrol/BUILD

```diff
@@ -0,0 +1,34 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "doc.go",
+        "register.go",
+        "zz_generated.deepcopy.go",
```

Comment:
`hack/update-bazel.sh` thinks "types" belongs here too.  See that and other complaints at https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/77048/pull-kubernetes-verify/1138826765037211648